auth.go

```diff
@@ -30,8 +30,21 @@ func (me *PlainAuth) Response() string {
 	return fmt.Sprintf("\000%s\000%s", me.Username, me.Password)
 }
 
+// CertAuth for RabbitMQ-auth-mechanism-ssl.
+type CertAuth struct {
```

Comment:
Can we rename this `ExternalAuth` to match the RabbitMQ Java library naming convention?